Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -734,6 +734,8 @@ void ExecuteCommand(u32 command_0, u32 command_1, u32 command_2, u32 output_addr
     INFO_LOG(DVDINTERFACE, "DVDLowReadDiskID");
     command_handled_by_thread = ExecuteReadCommand(0, output_address, 0x20, output_length, false,
                                                    reply_type, &interrupt_type);
+    // It doesn't fail but rather reads 0x00 instead?
+    interrupt_type = INT_TCINT;
```

Comment:
You should have a comment saying that we set the out buffer to zeroes in the IOS DI code (unless you want to manually set it to zeroes here).

Is it intentional that the error code gets set even though the interrupt type indicates success? (ExecuteReadCommand both sets an error code and sets the interrupt type to INT_DEINT, but you only undo setting the interrupt type. Maybe it would be better to have a way to make ExecuteReadCommand not set those things rather than having to undo it?)